extra-admin/throttled/autobuild/build

```
@@ -0,0 +1,26 @@
+abinfo "Installing all scripts and configs ..."
+mkdir "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled -p
```

Every command needs to add verbosity if possible. For this case:

```suggestion
mkdir -pv "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled
```
...